packages/labs/motion/src/view-transition-update.ts

```diff
@@ -0,0 +1,48 @@
+import {LitElement} from 'lit';
+
+type Constructor<T> = new (...args: any[]) => T;
+
+/** Extend a LitElement to use view transitions (if available) on DOM updates */
+export const ViewTransitionUpdate =
+    <T extends Constructor<LitElement>>(superClass: T) => {
+    class LitElementTransition extends superClass {
+        private lifecycleRenderTransition: ViewTransition;
+  
+        async performUpdate() {
+            if (!(document as DocumentWithTransition).startViewTransition)
+                return await super.performUpdate();
```

Comment:
run `npm run format` from the monorepo root

Reply:
Done in f1b75c87f7cef8ed6c2db7e6ac44fe3facbb2c04